domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/ReviewSchedule.kt

```diff
@@ -10,8 +10,7 @@ import java.util.UUID
 data class ReviewSchedule(
   val reference: UUID,
   val prisonNumber: String,
-  val earliestReviewDate: LocalDate,
-  val latestReviewDate: LocalDate,
+  val reviewScheduleWindow: ReviewScheduleWindow,
```

Comment:
It made sense to use the `ReviewScheduleWindow` class for these 2 fields, and to move the `ReviewScheduleWindow` class here (below)